data/json/recipes/recipe_deconstruction.json

```diff
@@ -2043,32 +2043,32 @@
   {
     "result": "fork",
     "type": "uncraft",
-    "time": "1 m 30 s",
+    "time": "30 s",
     "qualities": [ { "id": "HAMMER", "level": 1 } ],
     "components": [ [ [ "scrap", 1 ] ] ]
   },
   {
     "result": "glass",
     "type": "uncraft",
-    "time": "30 s",
+    "time": "1 s",
```

Comment:
the glass is 20ml smaller than the amount of glass shards would suggest.

Reply:
haha, sure.